### DISCUSSION TITLE
「开源之史」系列之三/六：从两场改变软件格局的运动讲起 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#77] [Announcements]
# 「开源之史」系列之三/六：从两场改变软件格局的运动讲起 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/04-01-free-software-and-os-movement/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
不对哪

> ...从两场改变软件格局的运动讲起

软件格局, 而不是 FLOSS 软件的话,
第一个更加重要的运动难道不应该是商业软件诞生?
要不是 Bill 先发明了软件许可证, RMS 也就不会丧失那么多小伙伴,
也就不会气到原地爆炸, 而灵感炸出一个 GNU/GPL 来;

后来的 OSI 只能说是自然发生的粘合剂,
在 盘右辟开了天地后,
中间当然也得有连接物...

当然, 同意作者提出的文化核心说,
提炼出来的软件四自由,
其实也是吻合其它社会场景中,

是否可以类比设想, 为什么 开源在中国 总是变形?
可能基于的文化背景中, 本身就没有满足那4个自由的文化基础?

当然, 这是绝对的雷区, 如果想不明白, 并给出浅显的比喻前, 嫑涉及这方面的深入...

不对, 作者必须先深入, 才能探知基本全貌, 然后,才能合理退出,
给出一个普适意义的可用途径...

**Reply by lijiangsheng1** [reply]
我应该在《开源之史》之外再接着写，《哪些文化需要习惯了本土的人们需要努力学习的》这样的文章，应该有：

0. 学习个人主义，一切围绕自身的主张做起。
1.  树立全球观念，中国之外的世界也很大。
2. 凡事亲力亲为，承认别人和承认自己一样重要
3. 不迷信权威，不顺从权威
4.  。。。。。。。
